deps/rabbitmq_stream/include/rabbit_stream_metrics.hrl

```diff
@@ -1,2 +1,94 @@
+-include_lib("rabbit/include/rabbit_global_counters.hrl").
+
 -define(TABLE_CONSUMER, rabbit_stream_consumer_created).
 -define(TABLE_PUBLISHER, rabbit_stream_publisher_created).
+
+-define(STREAM_DOES_NOT_EXIST, ?NUM_PROTOCOL_COUNTERS + 1).
+-define(SUBSCRIPTION_ID_ALREADY_EXISTS, ?NUM_PROTOCOL_COUNTERS + 2).
+-define(SUBSCRIPTION_ID_DOES_NOT_EXIST, ?NUM_PROTOCOL_COUNTERS + 3).
+-define(STREAM_ALREADY_EXISTS, ?NUM_PROTOCOL_COUNTERS + 4).
+-define(STREAM_NOT_AVAILABLE, ?NUM_PROTOCOL_COUNTERS + 5).
+-define(SASL_MECHANISM_NOT_SUPPORTED, ?NUM_PROTOCOL_COUNTERS + 6).
+-define(AUTHENTICATION_FAILURE, ?NUM_PROTOCOL_COUNTERS + 7).
+-define(SASL_ERROR, ?NUM_PROTOCOL_COUNTERS + 8).
+-define(SASL_CHALLENGE, ?NUM_PROTOCOL_COUNTERS + 9).
+-define(SASL_AUTHENTICATION_FAILURE_LOOPBACK, ?NUM_PROTOCOL_COUNTERS + 10).
+-define(VHOST_ACCESS_FAILURE, ?NUM_PROTOCOL_COUNTERS + 11).
+-define(UNKNOWN_FRAME, ?NUM_PROTOCOL_COUNTERS + 12).
+-define(FRAME_TOO_LARGE, ?NUM_PROTOCOL_COUNTERS + 13).
+-define(INTERNAL_ERROR, ?NUM_PROTOCOL_COUNTERS + 14).
+-define(ACCESS_REFUSED, ?NUM_PROTOCOL_COUNTERS + 15).
+-define(PRECONDITION_FAILED, ?NUM_PROTOCOL_COUNTERS + 16).
+-define(PUBLISHER_DOES_NOT_EXIST, ?NUM_PROTOCOL_COUNTERS + 17).
+
+-define(PROTOCOL_COUNTERS,
+        [
+         {
+          stream_error_stream_does_not_exist, ?STREAM_DOES_NOT_EXIST, counter,
```

Comment:
If these are counters, then Prometheus format requires them to end with `_total` : 

> Note that an accumulating count has total as a suffix, in addition to the unit if applicable.

https://prometheus.io/docs/practices/naming/